service/src/registry.ts

```diff
@@ -448,16 +453,28 @@ export class Registry {
         'outputBlobContainerUri': '<output container Uri given as parameter>',
         'storageAuthenticationType': 'IdentityBased'
       }
+      ```
+
+      If a `userAssignedIdentity` is provided, the following additional property shall be in the request body:
+      ```Node
+      "identity": {
+        "userAssignedIdentity": <resource ID for user assigned managed identity given as a parameter>
+      }
       ```]*/
-      const path = '/jobs/create' + endpoint.versionQueryStringLimitedAvailability();
+      const path = '/jobs/create' + endpoint.versionQueryString();
       const httpHeaders = {
         'Content-Type': 'application/json; charset=utf-8'
       };
       const importRequest = {
         'type': 'import',
         'inputBlobContainerUri': inputBlobContainerUri,
         'outputBlobContainerUri': outputBlobContainerUri,
-        'storageAuthenticationType': 'IdentityBased'
+        'storageAuthenticationType': 'IdentityBased',
+        ...(typeof doneOrIdentity === 'string' && {
```

Comment:
Well, I'll let you explain this to me.

Reply:
Where did you find this?

Reply:
https://amberley.dev/blog/2020-09-07-conditionally-add-to-array-or-obj/#conditionally-add-a-property-to-an-object